libs/components/pages/src/lib/modules/link-list/link-list.module.ts

```diff
@@ -1,23 +1,10 @@
-import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
-import { SkyWaitModule } from '@skyux/indicators';
-import { SkyAppLinkModule, SkyHrefModule } from '@skyux/router';
-import { SkyThemeModule } from '@skyux/theme';
-
-import { LinkAsModule } from '../link-as/link-as.module';
 
+import { SkyLinkListItemComponent } from './link-list-item.component';
 import { SkyLinkListComponent } from './link-list.component';
 
 @NgModule({
-  declarations: [SkyLinkListComponent],
-  exports: [SkyLinkListComponent],
-  imports: [
-    CommonModule,
-    SkyAppLinkModule,
-    SkyHrefModule,
-    SkyThemeModule,
-    SkyWaitModule,
-    LinkAsModule,
-  ],
+  exports: [SkyLinkListComponent, SkyLinkListItemComponent],
+  imports: [SkyLinkListComponent, SkyLinkListItemComponent],
```

Comment:
The story mentions exporting the `SkyLinkListModule` from the `SkyPageModule`. Is that feasible?

Reply:
Good catch. I've added the new components to `SkyPageModule`'s exports.